### PR TITLE
Return JSON Objects

### PR DESCRIPTION
**Pull request by SeamusMacRat**
First few bits of streaming objects.

Return JSON Objects

NB: First of a few requests for this branch, so please leave the branch open after the pull request.

### CODE REVIEW

**Comment by SeamusMacRat** [comment]
Streaming removed as we can use the multipart upload

File: RESTore/ExecutionContext.cs [hunk 1]
```diff
@@ -52,6 +52,11 @@ public class ExecutionContext
         /// </summary>
         private HttpClient _httpClient;
 
+        /// <summary>
+        /// 
+        /// </summary>
+        private HttpWebRequest _webRequest;
+
```
Comment (delahayeoglafa) [inline RESTore/ExecutionContext.cs]: empty summary here Kev

File: RESTore/ExecutionContext.cs [hunk 2]
```diff
@@ -425,9 +432,10 @@ public async Task MapCall()
         /// <returns>A task representing the executed call.</returns>
         private async Task<TimedResponse> ExecuteCall()
         {
+            HttpResponseMessage response = null;
```
Comment (delahayeoglafa) [inline RESTore/ExecutionContext.cs]: do we need to set response as null here? Cant we just have HttpResponseMessage response = await _httpClient.SendAsync(BuildRequest()); ??? I cant see it being used outside of the scope of this

File: RESTore/ExecutionContext.cs (hunk 1 above)
Comment (SeamusMacRat) [reply RESTore/ExecutionContext.cs]: Added as part of the performance pull request :)

File: RESTore/ExecutionContext.cs (hunk 2 above)
Comment (SeamusMacRat) [reply RESTore/ExecutionContext.cs]: Changed on the performance branch, I believe

**Pull request merged**